File: Python3/encrypt.py
def encrypt(initial):
  """ Use : encrypt("message ")
  => 'nvhhztv'
  """
  mylist = []
  
  initial = initial.lower()

  for char in initial:
    mylist.append(ord(char) - 97)
  
  output = ""
  alphabet = dict(zip(range(0, 26), string.ascii_lowercase))
  
  for char in mylist:
    image = str(alphabet[25 - char])
    output += image
  
  return output

File: js/encrypt.js
String.prototype.encrypt = function() {
	const alphabet = 'abcdefghijklmnopqrstuvwxyz '.split('')
	let output = "";
	for (let i of this) {
		output += alphabet[25 - alphabet.indexOf(i.toLowerCase())]
	}
	return output;
}
module.exports = text => text.encrypt()
